src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from "react";
+import LoginModal from '../Modals/LoginModal/LoginModal'
+import { FaAlignRight } from 'react-icons/fa';
+import { logoutUser } from '../../actions/userHelpers'
+import { useAuthDataContext } from '../../actions/AuthDataProvider'
+import "./Navbar.css";
+
+const Navbar = () => {
+  const [ toggle, setToggle ] = useState(0);
+  const [ showModal, setShowModal ] = useState(false);
+  const { current_user, logged_in, onLogout } = useAuthDataContext();
+
+  const handleLogout = () => {
+    loginModalHide()
+    logoutUser()
+    onLogout()
+  }
+
+  function loginModalShow() {
+    setShowModal(true)
+  }
+
+  function loginModalHide() {
+    setShowModal(false)
+  }
+
+  const li = [
```

Comment:
Lets remove these and use the Link tag from react router

Reply:
Not sure what you mean, example code?